FrontPage/mobile-services.json

```diff
@@ -0,0 +1,29 @@
+{
```

Comment:
Should the `mobile-service.json` file not have sample data.

Reply:
for I would prefer to leave it as it is Jim